Setup/Shared/Get-FileInformation.ps1

```diff
@@ -34,19 +33,26 @@ Function Get-FileInformation {
                 return
             }
 
-            <#
-TODO: Fix this code. Clearly didn't finish it.
-        $shellApplication = New-Object -ComObject "Shell.Application"
-
-        if (-not($shellApplication)) {
-            "Failed to create 'Shell.Application' COM Object. This can lead to issues with validation of the script." | Receive-Output -ForegroundColor Red
-            exit
-        }
-
-        $fileItem = Get-Item $File
-        $shellFolder = $shellApplication.NameSpace($fileItem.Directory.FullName)
-        $subject = $shellFolder.GetDetailsOf($shellFolder.ParseName($fileItem.Name), 22)
-#>
+            <# TODO: Still need more testing (Making sure it produces same properties)
+                try {
+                    $shellApplication = New-Object -ComObject "Shell.Application"
+                } catch {
+                    "Failed to create 'Shell.Application' COM Object. This can lead to issues with validation of the script." | Receive-Output -ForegroundColor Red
+                    break
+                }
+                # One $fileItem Example:
+                # $localPackageChildItems = Get-ChildItem -Path "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Installer" -Recurse | Where-Object { $_.Property -eq "LocalPackage" };
+                # $File = [IO.FileInfo]$localPackageChildItems[1].GetValue("LocalPackage");
+                # $fileItem = Get-Item $File
+                $fileItem = Get-Item $File
```

Comment:
What is this code about? The PR is regarding Get-ExchangeBuildVersionInformation.ps1 and doesn't provide anything about what we are fixing in here. 

Secondly, setup code does NOT call Get-ExchangeBuildVersionInformation. These should not be in the same PR anyways.